compliance_checker/cf/cf.py

```diff
@@ -2513,7 +2516,7 @@ def check_cell_boundaries(self, ds):
             if boundary_variable_name not in ds.variables:
                 valid = False
                 reasoning.append("Boundary variable {} referenced by {} not "
-                                 "found in dataset variables".format(boundary_variable.name,
+                                 "found in dataset variables".format(boundary_variable_name,
```

Comment:
PS: there is another one at ``./compliance_checker/cf/cf.py:2820:25: F821 undefined name 'boundary_variable'``